serverless.yml

```diff
@@ -0,0 +1,38 @@
+service: flask-api-lambda
+
+provider:
+  name: aws
+  runtime: python3.12.2
+  region: us-east-1
+
+functions:
+  api:
+    handler: wsgi_handler.handler
+    events:
+      - http:
+          path: /{proxy+}  
+          method: any
+
+plugins:
```

Comment:
Is there a way to add comments in a yml file? That would help with understanding this stuff.